transformers_sklearn/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# This notebook currently requires the latest developer version of cleanlab: pip install git+https://github.com/cleanlab/cleanlab.git
+ipywidgets==8.0.2
+numpy>=1.22.0
+pandas==1.5.0
+scikit_learn==1.0.2
```

Comment:
are these 3 really needed or is the versions installed with cleanlab already ok?

Reply:
versions installed with cleanlab are alright, these are just specified as versions used to run the current example - can remove if we think are not necessary to list

Reply:
ya lets remove them, this'll presumably be stable with all cleanlab versions (and their dependencies) going forward